syncd/syncd_notifications.cpp

```diff
@@ -92,19 +92,122 @@ void redisPutFdbEntryToAsicView(
         return;
     }
 
-    if ((fdb->event_type == SAI_FDB_EVENT_AGED)||(fdb->event_type == SAI_FDB_EVENT_FLUSHED))
+    if (fdb->event_type == SAI_FDB_EVENT_AGED)
     {
-        if (fdb->event_type == SAI_FDB_EVENT_AGED)
+        SWSS_LOG_DEBUG("remove fdb entry %s for SAI_FDB_EVENT_AGED",key.c_str());
+        g_redisClient->del(key);
+        return;
+    }
+
+    if (fdb->event_type == SAI_FDB_EVENT_FLUSHED)
+    {
+        sai_vlan_id_t vlan_id = fdb->fdb_entry.vlan_id;
+        sai_object_id_t port_oid = 0;
+        int port_oid_found = 0;
+        int attr_num = fdb->attr_count;
```

Comment:
extra variable not needed, use direct attr_count in line 109

Reply:
optimized as suggested.